samples/gibbs_ensemble/gibbs_ensemble.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2013-2019 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Client class containing the methods used for communicating and manipulating the system
+MessageID class for communication identifiers
+"""
+
+from enum import unique, Enum
+from multiprocessing import Process
+import os
+import numpy as np
+
+
+@unique
+class MessageID(Enum):
+    # Message identifiers
+    ENERGY = 0
+    END = 1
+    MOVE_PART = 2
+    MOVE_PART_REVERT = 21
+    CHANGE_VOLUME = 3
+    CHANGE_VOLUME_REVERT = 31
+    PART_ADD = 4
+    PART_ADD_REVERT = 41
+    PART_REMOVE = 5
+    PART_REMOVE_REVERT = 51
+    INFO = 6
```

Comment:
do these enum values have a special meaning? if not consider using automatic generation of the values via ```enum.auto()```